prover/proof_submitter/util.go

```diff
@@ -24,7 +24,7 @@ var (
 // isSubmitProofTxErrorRetryable checks whether the error returned by a proof submission transaction
 // is retryable.
 func isSubmitProofTxErrorRetryable(err error, blockID *big.Int) bool {
-	if strings.HasPrefix(err.Error(), "L1_NOT_ORACLE_PROVEN") || !strings.HasPrefix(err.Error(), "L1_") {
+	if strings.HasPrefix(err.Error(), "L1_NOT_ORACLE_PROVER") || !strings.HasPrefix(err.Error(), "L1_") {
```

Comment:
```suggestion
	if !strings.HasPrefix(err.Error(), "L1_") {
```
Since oracle proof is allowed not to be the first proof, we can remove the condition here?

Reply:
this function is also used if not oracle proof though, its an existing method used in valid_proof_submitter